### DISCUSSION TITLE
Change device to different Garmin device (Forerunner 945 default)

### DISCUSSION THREAD

**Discussion by manimohans** [philosowaffle/peloton-to-garmin#647] [Q&A] [answered]
Hello, I am running the app on Docker using Mac. 
I tried to change device by creating my own deviceInfo.xml as follows, and I pointed to it via the webUI. 

Still writes out to Garmin as 945 - what am I missing? 

``` XML 
<Creator xsi:type="Device_t">
    <Name>Forerunner 265</Name>
    <UnitId>1234536286</UnitId>
    <ProductID>4367</ProductID>
    <Version>
        <VersionMajor>19</VersionMajor>
        <VersionMinor>18</VersionMinor>
        <BuildMajor>0</BuildMajor>
        <BuildMinor>0</BuildMinor>
    </Version>
</Creator> 
```

**Comment by philosowaffle** [comment, marked as answer]
Please try updating to the new release [4.2.0](https://github.com/philosowaffle/peloton-to-garmin/releases/tag/v4.2.0).  This provides a way to configure custom devices using the UI which should be a better, less error prone experience.
